content/src/controller/Controller.ts

```diff
@@ -90,6 +94,41 @@ export class Controller {
     res.send(maskedEntities)
   }
 
+  async getActiveEntities(
+    req: express.Request<unknown, unknown, { ids: string[]; pointers: string[] }>,
+    res: express.Response
+  ): Promise<void> {
+    // Method: POST
+    // Path: /entities/active
+    // Body: { ids: string[], pointers: string[]}
+
+    const ids: EntityId[] = req.body.ids
+    const pointers: Pointer[] = req.body.pointers
+
+    console.log(JSON.stringify(req.body))
+    // Validate pointers or ids are present, but not both
+    if (
+      (ids && ids.length > 0 && pointers && pointers.length > 0) || // both present
+      ((!ids || ids.length == 0) && (!pointers || pointers.length == 0)) // none present
+    ) {
+      res.status(400).send({ error: 'ids or pointers must be present, but not both' })
+      return
+    }
+
+    let entities: Entity[] = []
+    if (ids && ids.length > 0) {
+      entities = await this.components.deployer.getEntitiesByIds(ids)
+    } else if (pointers && pointers.length > 0) {
+      for (const type of Object.values(EntityType)) {
+        entities = [...entities, ...(await this.components.deployer.getEntitiesByPointers(type, pointers))]
```

Comment:
This method should be `await this.components.deployer.getEntitiesByPointers(pointers)`

Reply:
changed cache to be by pointer (removing entity type)